core/src/apps/ontology/serialize.py

```diff
@@ -95,7 +94,8 @@ def _serialize_transfer_payload(transfer: OntologyTransfer):
     native_call = build_native_call("transfer", [[struct]], contract)
 
     # 9 is the maximum possible length of the uvarint prefix
-    ret = bytearray(len(native_call) + 9)
+    # ret = bytearray(len(native_call) + 9)
```

Comment:
Why this change?

Reply:
should've added a comment. with the original code both tests in test_msg_ontology_sign_tx fail. I need to investigate why and fix it properly.